### DISCUSSION TITLE
How to set the maximum number of authors to 6？

### DISCUSSION THREAD

**Discussion by SVT-Yang** [retorquere/zotero-better-bibtex#2798] [Q&A] [answered]
Hell0! I wonder how to set the maximum number of authors to 6. If there are more than 6 authors, add "the others" at the end.

**Comment by retorquere** [comment, marked as answer]
I had to fix a bug first to get this to work again, so in the latest release:

```
creators(n=[1,6]) + creators(n=7).replace(/.+/, theothers)
```

(there's not really a reason to have `'the others'` as the space will get removed anyhow.
